packages/react/src/themes/teams/components/Embed/embedVariables.ts

```diff
@@ -0,0 +1,13 @@
+export interface EmbedVariables {
```

Comment:
Please, for consistency, add these new styles and variables in `themes/types.ts` in:
`ThemeComponentStylesInput`
`ThemeComponentStylesPrepared`
`ThemeComponentVariablesInput`
`ThemeComponentVariablesPrepared`

Reply:
Also added for `Tree` component 👍